### DISCUSSION TITLE
Attribute Page with extensibleEnumOption overload

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#798] [Q&A] [answered]
![Screenshot from 2022-05-04 11-42-00](https://github.com/atrocore/atrocore/assets/30953478/afb3ccdc-c30b-4562-879f-b8d7f697308e)
after openning attribute page 50-70k long extensible enum's, page got frozen

**Comment by rratsun** [comment, marked as answer]
Try to update to the latest version of Core and PIM

**Reply by zssvaidar** [reply]
Have been testing new versions, We have ran pim with latest core and pim, no good

Remember you suggesting on https://github.com/atrocore/atrocore/discussions/715 "For now, I recommend to you to create new Entity and create a relationship between Product and your new Entity and use this field for your needs." -> problems with attribute lists that we had had been resolved in new version and working great

Only the last problem we've encountered involving lists being too long to load onto front-end given in this picture
![Screenshot from 2022-05-11 15-57-36](https://github.com/atrocore/atrocore/assets/30953478/180c5065-c33d-429c-9b56-607ffbcabe01)

Attribute page with long list of extensibleEnum's use up more than 1.5gb of ram memory and make all site pages freez i believe

thank you

**Reply by rratsun** [reply]
Which versions do you have now? For core and pim.

**Reply by zssvaidar** [reply]
pim 1.9.17
core 1.6.13

![image](https://github.com/atrocore/atrocore/assets/30953478/fcbe657f-5d6f-4622-8cbc-1fed001d0d41)
![Screenshot from 2022-05-04 11-42-00](https://github.com/atrocore/atrocore/assets/30953478/2090597a-a12f-4b64-b5ec-d0587c6935da)

**Reply by rratsun** [reply]
ok. thanks. we will check.

**Comment by asapko** [comment]
This has been fixed, previously All options where shown on attribute detail page, now we have a button "Show More"
